crates/ty_python_semantic/resources/mdtest/generics/legacy/classes.md

```diff
@@ -392,6 +392,33 @@ class C(tuple[T, U]): ...
 reveal_type(C((1, 2)))  # revealed: C[int, int]
 ```
 
+### Upcasting a `tuple` to its `Sequence` supertype
+
+This test is taken from the
+[typing spec conformance suite](https://github.com/python/typing/blob/c141cdfb9d7085c1aafa76726c8ce08362837e8b/conformance/tests/tuples_type_compat.py#L133-L153)
+
+```toml
+[environment]
+python-version = "3.11"
```

Comment:
Out of curiosity, which part needs 3.11?

Reply:
`t2: tuple[int, *tuple[str, ...]]` is invalid syntax on Python <=3.10 -- starred elements are never allowed in annotations on older versions of Python. On Python <=3.10, you would have to use `t1: tuple[int, Unpack[tuple[str, ...]]]`, but we don't support `Unpack` yet

Reply:
Starred tuples, yes! That paged out of my memory quickly :grimacing: